src/static_ros_publisher.cpp

```diff
@@ -64,7 +64,7 @@ void StaticRosPublisher::run()
         //auto done = std::chrono::high_resolution_clock::now();
 
         RCLCPP_INFO(rclcpp::get_logger("rclcpp_imu_identification_data"), "Publishing static data: '%s' '%s' flash_counter = '%d' ",
-                    m_message.firmware_revision.c_str(), m_message.firmware_date.c_str(), m_message.flash_counter);
+                    m_message.firmware_revision.c_str(), m_message.firmware_date.c_str(), m_message.serial_number);
```

Comment:
Something is strange here.

First of all let's also adapt the new name from static data to identification data. So there should actually be Publishing identification data. The same goes for the file name, instead of static_ros_publisher you could rename it to imu_identification_data_publisher, check this for all file names that are using "static_data" and change to imu_identification_data.

Secondly, flash_counter should not be there in the comment. 
This printing needs some re-work.

Let's change it to something like:

"Publishing IMU identification data: \nserial number: '%d'\nproduct id: '%d'\nfirmware data: '%s'\nfirmware revision: '%s'\ngyroscope measurement range: '%s'"